tests/Ormolu/Diff/TextSpec.hs

```diff
@@ -45,7 +45,7 @@ stdTest name pathA pathB = it name $ do
   expectedDiffText <-
     parseRelFile expectedDiffPath
       >>= readFileUtf8 . toFilePath . (diffOutputsDir </>)
-  let Just actualDiff = diffText inputA inputB "TEST"
+  Just actualDiff <- pure $ diffText inputA inputB "TEST"
```

Comment:
Why like this?

Reply:
Otherwise, I get (but only on GHC 9.2)
```haskell
tests/Ormolu/Diff/TextSpec.hs:48:7: error: [-Wincomplete-uni-patterns, -Werror=incomplete-uni-patterns]
    Pattern match(es) are non-exhaustive
    In a pattern binding:
        Patterns of type ‘Maybe TextDiff’ not matched: Nothing
   |
48 |   let Just actualDiff = diffText inputA inputB "TEST"
   |       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
which I found a bit confusing, as `-Wincomplete-uni-patterns` does not seem to be enabled for the test suite, but it was a boring enough change so I just went with it.